kolibri/core/assets/src/views/connection-snackbars/index.vue

```diff
@@ -8,18 +8,21 @@
       :actionText="$tr('tryNow')"
       :backdrop="true"
       @actionClicked="tryToReconnect"
+      :key="$tr('tryNow')"
```

Comment:
To prevent vue from reusing the snackbar, to trigger the transition.

Reply:
for future reference: https://vuejs.org/v2/api/#key